www/src/components/Toc.scss

```diff
@@ -0,0 +1,38 @@
+.pgn-doc__toc {
+  position: sticky;
```

Comment:
I'm wondering if it would make sense to start using the new `Sticky` component here to manage the sticky positioning of the ToC?

Reply:
yes, updated with Sticky usage